### DISCUSSION TITLE
Inconsistent colouring of mod multipliers depending on context

### DISCUSSION THREAD

**Discussion by ghost** [ppy/osu#20975] [Q&A] [answered]
(Not sure if this is intended or not.)
In the song selection menu, mod multipliers over 1.00x are coloured green, and mod multipliers below 1.00x are coloured red
However in the mod selection menu, mod multipliers over 1.00x are coloured red, and mod multipliers below 1.00x are coloured green (the opposite of the song selection menu).

![song selection menu](https://user-images.githubusercontent.com/102317848/198457029-0d5312a2-568a-4f9c-9d34-6e64c2a5a875.png)
![mod selection menu](https://user-images.githubusercontent.com/102317848/198457940-a145fc7a-8523-48e8-80a7-383b311f79b0.png)

Comparison:
|   | Less than 1.00x multiplier | 1.00x multiplier | More than 1.00x multiplier |
| - | - | - | - |
| Mod selection menu | ![image](https://user-images.githubusercontent.com/102317848/198460590-62928f21-995a-404d-a383-272f5017bf4d.png) | ![image](https://user-images.githubusercontent.com/102317848/198460439-adb41b9a-39b2-41ea-9efa-5c7da1f2154f.png) | ![image](https://user-images.githubusercontent.com/102317848/198459822-83e28ffa-16d1-4d30-8e26-f5ea40d57e6d.png) |
| Song selection menu | ![image](https://user-images.githubusercontent.com/102317848/198461165-e5fea159-f892-46da-b116-c42050685e40.png) | ![image](https://user-images.githubusercontent.com/102317848/198460971-cb4dc094-d76e-495d-8eca-b0cdd2bad594.png) | ![image](https://user-images.githubusercontent.com/102317848/198460788-628a5f46-9148-46e4-af46-a6bd9de8f207.png) |

**Comment by peppy** [comment, marked as answer]
The one at song select is an outdated UI element and will eventually match the other colouring.
